docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst

```diff
@@ -0,0 +1,112 @@
+Add the power of Apache Flink® to Apache Kafka® for stateful stream processing
+==============================================================================
+
+Whether it's an event-driven application or a data pipeline, Apache Flink® is an excellent choice due to its ability to perform computations at in-memory speed and at any scale. 
+We can use Apache Kafka® as a highly-available, fault-tolerant platform and leverage the power of Flink's stateful data transformations alongside. This example shows how to set up an Aiven for Apache Kafka
+with an Aiven for Apache Flink integration using the `Aiven Terraform Provider <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_.
+
+Describe the setup
+------------------
+
+The following Terraform script stands up both Apache Kafka and Apache Flink services, creates the service integration and  related resources. The parameters and configurations will vary for your case and you can find
+links to the advanced parameters as a reference at the end of this document.
+
+
+``services.tf`` file:
+
+.. code:: bash
+
+  resource "aiven_flink" "flink" {
+   project      = data.aiven_project.dev-advocates.project
```

Comment:
Please check what standard indentation is for these files, one space doesn't look quite right, I think it should be 2 but let's do whatever the main terraform project docs use!